_src/css/_section.scss

```diff
@@ -48,15 +85,37 @@
     &__text {
       color: white;
       font-family: pt mono;
-      line-height: 25px;
+      line-height: 28px;
       font-size: 16px;
       text-align: justify;
+
+      @media screen and (max-width: $web-up) {
+        padding: 50px;
+        line-height: 45px;
+        }
+          
+      @media screen and (max-width: $tablet-up) {
+        padding: 0;
+        line-height: 30px;
+        font-size: 12px;
+      }
+
+      @media screen and (max-width: $mobile-up) {
+        line-height: 20px;
+        font-size: 11px;
+        padding: 0;
+      }
     }
 
     &__vector {
       position: absolute;
```

Comment:
@felipprodrigues great job here, but let's only display the `position: absolute` when the other media query is not activated. In other words, you must add that rule in another `@media` greater than the `$web-up` breakpoint.